### DISCUSSION TITLE
Editor mockups

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [Wohlstand/TheXTech#707] [Ideas]
I just had some mockup on how the editor could look
Groups could be added in the main top bar when the topbar item is selected
Also Godot doesn't support multiple focus but ideally tileset items would keep focus even if the toolbar has focus
Also I didn't have a settings button but that could contain level settings and file functions like new, save, and open

![image](https://github.com/Wohlstand/TheXTech/assets/2453707/f84b28c9-bd9b-44bc-bc88-6e77dde519d6)
![image](https://github.com/Wohlstand/TheXTech/assets/2453707/a5db1e3e-4e8d-421d-bc03-2c2dd0712a01)
![image](https://github.com/Wohlstand/TheXTech/assets/2453707/f1f0665e-7e04-4e9c-95b5-603815b43253)
![image](https://github.com/Wohlstand/TheXTech/assets/2453707/0b067eb1-4a4e-4e1b-acb4-00984218f2d4)
![image](https://github.com/Wohlstand/TheXTech/assets/2453707/7040de8d-a592-41a5-b1fe-ceece623cf19)

**Comment by Shadowblitz16** [comment]
@ds-sloth did you see this?

I actually forgot about it.

**Reply by ds-sloth** [reply]
I did see this and I'm keeping it in mind. Major changes to the editor just haven't been on the top of our list.

**Reply by Shadowblitz16** [reply]
K cool
